Use Case: Watering layer (#222)
ElektraInitiative/PermaplanT: doc/usecases/layers/watering_layer.md

> 

Reviewer: ```suggestion
```

> # Use Case: Watering Management

Reviewer: Name should be same as file name:

```suggestion
# Use Case: Watering Layer
```

## Summary

- **Scope:** Watering Layer
- **Level:** User Goal
- **Actors:** App User
- **Brief:** The user can log watering events for individual plants or the entire garden, including rainwater, to track watering history and improve plant care.

> - **Status:** Draft

Reviewer: ```suggestion
- **Status:** Assigned
```

> - **Assignee:** Ramzan

Reviewer: That means you also want to implement it.

## Scenarios

- **Precondition:**
 - The user has opened the app and has selected the watering layer.
- **Main success scenario:**

>  - The user logs a watering event for an individual plant or a specific garden area.

Reviewer: ```suggestion
 - The user logs a watering event:
   - by selecting individual plants
   - by brushing over the map
   - for all parts of the map, which are open-air (area can be defined by brushing where roofs etc. are)
```
